### ISSUE TITLE
Retrieving data from external API

### ISSUE THREAD

**Issue by mclouvem** [tomatau/breko-hub#11]
HI @tomatau,

What is your recommendation to retrieve data from an external API? May I ask you for some example? 

I'm making some tests, but I couldn't get these data inside my thumb component.

**Comment by tomatau** [comment]
The redux-promise-middleware is bundled in already, so any promise for a request would work there. Although it's totally up to you how you prefer to make requests -- there are many libraries out there to ease this for redux.

**Comment by mclouvem** [comment]
Hi @tomatau,
Thanks, I've made the request to the API and I got external data inside state. Also, I've used react-redux Connect inside 'App' component
```Javascript
export default function mapStateToProps(state) {
  return {
    ...state,
  }
}

export default connect(mapStateToProps, null)(App)
```

Using React DevTools, I can see all state inside de App component. I'm trying to pass the state to FooRoute, but I'm making some mistake, because there is not state data inside FooRoute component. I know it is not an issue, but I'll really appreciate some help to the it to work.

Thanks!
